framework/src/outputs/Console.C

```diff
@@ -455,6 +453,49 @@ Console::writeTimestepInformation(bool output_dt)
   }
 }
 
+std::string
+Console::formatTime(const Real t) const
+{
+  std::ostringstream oss;
+  if (_time_format == "plain" || _time_format == "second")
+  {
+    if (_precision > 0)
+      oss << std::setw(_precision) << std::setprecision(_precision) << std::setfill('0')
+          << std::showpoint;
+    if (_scientific_time)
+      oss << std::scientific;
+    oss << t;
+    if (_time_format == 1)
```

Comment:
use enum

Reply:
Yeah, I was using `unsigned int _time_format` initially, but that makes the setter interface harder.